samza-core/src/main/java/org/apache/samza/runtime/ApplicationRunnerMain.java

```diff
@@ -34,19 +34,48 @@
  */
 public class ApplicationRunnerMain {
   // TODO: have the app configs consolidated in one place
-  private static final String STREAM_APPLICATION_CLASS_CONFIG = "app.class";
+  public static final String STREAM_APPLICATION_CLASS_CONFIG = "app.class";
+
+  public static class ApplicationRunnerCommandLine extends CommandLine {
+    public OptionSpec operationOpt =
+        parser().accepts("operation", "The operation to perform; run, status, kill.")
+            .withRequiredArg()
+            .ofType(String.class)
+            .describedAs("operation=run")
+            .defaultsTo("run");
```

Comment:
Legacy support in JobRunner. It never required this parameter before and would always execute run(). 

Defaulting to run just results in cleaner code than making the argument optional, checking if it's missing, and then executing run()